test/test.js

```diff
@@ -0,0 +1,103 @@
+var server = require('../app/server');
+var assert = require('assert');
+var http = require('http');
+var request = require('superagent');
+var request2 = require('supertest');
+var expect = require('expect.js');
+var url = 'localhost:9000';
+
+describe('server', function () {
+    before(function (done) {
+        done();
+    });
+
+    // after(function (done) {
+    //     server.close();
+    //     done();
+    // });
+});
+
+describe('Testing attributes:', function () {
+    var id;
+
+   //getting all attributes
+    it('Getting all attributes', function(done) {
```

Comment:
Something sounds wrong here. "it Getting all attributes" isn't proper English :)

Read the output from the tests both when they pass and when they fail to make sure everything reads well.